Index: src/main/java/com/coolbank/model/User.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.coolbank.model;\r\n\r\npublic class User {\r\n}\r\n
===================================================================
diff --git a/src/main/java/com/coolbank/model/User.java b/src/main/java/com/coolbank/model/User.java
--- a/src/main/java/com/coolbank/model/User.java	
+++ b/src/main/java/com/coolbank/model/User.java	
@@ -1,4 +1,108 @@
 package com.coolbank.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+
+import java.util.HashMap;
+import java.util.UUID;
+
+@Entity
+@Table(name = "user")
 public class User {
+
+    private UUID idUser;
+
+    private String name;
+    private String surname;
+    private int yearOfBirth;
+    private int monthOfBirth;
+    private int dayOfBirth;
+    private HashMap<String, String> address;
+
+    private Account account;
+
+    private Card card;
+
+    public User() {
+    }
+
+    public User(UUID idUser, String name, String surname, int yearOfBirth, int monthOfBirth, int dayOfBirth, HashMap<String, String> address, Account account, Card card) {
+        this.idUser = idUser;
+        this.name = name;
+        this.surname = surname;
+        this.yearOfBirth = yearOfBirth;
+        this.monthOfBirth = monthOfBirth;
+        this.dayOfBirth = dayOfBirth;
+        this.address = address;
+        this.account = account;
+        this.card = card;
+    }
+
+    public UUID getIdUser() {
+        return idUser;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getSurname() {
+        return surname;
+    }
+
+    public void setSurname(String surname) {
+        this.surname = surname;
+    }
+
+    public int getYearOfBirth() {
+        return yearOfBirth;
+    }
+
+    public void setYearOfBirth(int yearOfBirth) {
+        this.yearOfBirth = yearOfBirth;
+    }
+
+    public int getMonthOfBirth() {
+        return monthOfBirth;
+    }
+
+    public void setMonthOfBirth(int monthOfBirth) {
+        this.monthOfBirth = monthOfBirth;
+    }
+
+    public int getDayOfBirth() {
+        return dayOfBirth;
+    }
+
+    public void setDayOfBirth(int dayOfBirth) {
+        this.dayOfBirth = dayOfBirth;
+    }
+
+    public HashMap<String, String> getAddress() {
+        return address;
+    }
+
+    public void setAddress(HashMap<String, String> address) {
+        this.address = address;
+    }
+
+    public Account getAccount() {
+        return account;
+    }
+
+    public void setAccount(Account account) {
+        this.account = account;
+    }
+
+    public Card getCard() {
+        return card;
+    }
+
+    public void setCard(Card card) {
+        this.card = card;
+    }
 }
